src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-user-details/rgw-user-details.component.html

```diff
@@ -68,6 +68,11 @@
                   </div>
                 </td>
               </tr>
+              <tr *ngIf="user.mfa_ids && user.mfa_ids.length">
```

Comment:
Let's benefit from the safe-navigation operator
```suggestion
              <tr *ngIf="user.mfa_ids?.length">
```